src/StackExchange.Redis/ConfigurationOptions.cs

```diff
@@ -792,21 +792,21 @@ private void DoParse(string configuration, bool ignoreUnknown)
 
         internal bool IsAzureEndpoint()
```

Comment:
Is there an easy way to make this `Lazy<>`, so it executes only once per `ConnectionMultiplexer` instance?

Reply:
This only happens during init/connect; I wouldn't be concerned about memoizing anything here - that would seem to add memory overhead for something that will never (or very rarely) be used again

Reply:
Makes sense. Will resolve